torch/distributed/checkpoint/optimizer.py

```diff
@@ -202,15 +202,15 @@ def load_sharded_optimizer_state_dict(
     storage_reader: dist_cp.StorageReader,
 ) -> STATE_DICT_TYPE:
     """
-    Loads a state_dict to be used in conjuntion with FSDP sharded optimizer state.
-    This is the current recommended way to checkpoint is FSDP
+    Loads a state_dict in conjuntion with FSDP sharded optimizer state.
```

Comment:
just for my learning, is this function specific to FSDP?

Reply:
Yes!